code/modules/mob/living/blood.dm

```diff
@@ -71,13 +65,18 @@
 				death()
 
 
-		// Without enough blood you slowly go hungry.
-		if(blood_volume < BLOOD_VOLUME_SAFE)
+		// Blood regens using food, more food = more blood.
+		if(blood_volume < BLOOD_VOLUME_NORMAL)
 			switch(nutrition)
-				if(300 to INFINITY)
+				if(450 to INFINITY)
```

Comment:
Use the NUTRITION_OVERFED defines for these, since those're the thresholds you're keying off anyway.